recipes/tttrlib/build.sh

```
@@ -0,0 +1,30 @@
+  export CONFIG_ARGS="-DCMAKE_FIND_FRAMEWORK=NEVER -DCMAKE_FIND_APPBUNDLE=NEVER"
+else
+  export CONFIG_ARGS=""
+fi
+
+cmake -S .. -B . \
+  -DCMAKE_CXX_COMPILER="${CXX}" \
+  -DCMAKE_INSTALL_PREFIX="$PREFIX" \
+  -DBUILD_PYTHON_INTERFACE=ON \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DBUILD_LIBRARY=ON \
+  -DPYTHON_VERSION=$(python -c 'import platform; print(platform.python_version())') \
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)